extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfoItemExtractor.java

```diff
@@ -3,59 +3,93 @@
 import org.schabi.newpipe.extractor.InfoItemExtractor;
 import org.schabi.newpipe.extractor.exceptions.ParsingException;
 import org.schabi.newpipe.extractor.localization.DateWrapper;
+import org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeCommentsInfoItemExtractor;
 import org.schabi.newpipe.extractor.stream.StreamExtractor;
+import org.schabi.newpipe.extractor.utils.Utils;
 
 import javax.annotation.Nullable;
 
 public interface CommentsInfoItemExtractor extends InfoItemExtractor {
 
     /**
-     * Return the like count of the comment, or -1 if it's unavailable
+     * Return the (approximate) like count of the comment, or -1 if it's unavailable<br/>
```

Comment:
improve the java doc:

first line should be summary

then description. move (approximate) to NOTE because it's only YouTube implementation, the interface doesn't require an approximation, we might understand that with this sentence